libraries/Microsoft.Bot.Builder/ShowTypingMiddleware.cs

```diff
@@ -110,7 +110,14 @@ private static async Task SendTypingActivityAsync(ITurnContext turnContext, Canc
                 Type = ActivityTypes.Typing,
                 RelatesTo = turnContext.Activity.RelatesTo,
             };
-            await turnContext.SendActivityAsync(typingActivity, cancellationToken).ConfigureAwait(false);
+
+            // sending the Activity directly on the Adapter avoids other Middleware and avoids setting the Responded
```

Comment:
This is a mistake. This will bypass logging / telemetry and other mechanisms. There's got to be a better way to do this.

Reply:
That is intentional. That’s the point of the fix.

Reply:
This is by design... We don't want these logged because they're an automated response that will flood your logs with essentially junk. It's an optional piece of middleware so if you want a version that logs then create one that logs.

Reply:
Ok. The crowd has spoken. Merged.